### PR TITLE
article on generate subsequences added

### LINKED ISSUE #476: Content: Generate all subsequences of a string
## 💥 Proposal

Given a string containing n characters, find out all the subsequences of the string.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

Please assign it to me.

### PR DESCRIPTION
**Pull request by gayatri517**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Both codes in python and c++ are written by me

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #476

### CODE REVIEW

**Comment by gayatri517** [comment]
Please review my PR @HarshCasper @iamrajiv @ankitaggarwal23 @atarax665 @gat786 @nidh-ish
Thank you

**Review by iamrajiv — APPROVED** [review]

**Comment by gayatri517** [comment]
@atarax665 please review my PR

File: docs/competitive-programming/generate-subsequences.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+slug: /competitive-programming/generate-subsequences
+---
+
+## Introduction
+
+Given a string containing n characters, find out all the subsequences of the string.
+
+## Explanation 
+
+We can do it by first taking an empty string. Initially, there will two seperate cases,
```
Comment (atarax665) [inline docs/competitive-programming/generate-subsequences.mdx:12]:
```suggestion
We can do it by first taking an empty string. Initially, there will be two separate cases, 
```

**Review by atarax665 — APPROVED** [review]

**Comment by gayatri517** [comment]
@atarax665 i have made the changes and resolved the conflicts kindly review it.

File: sidebars.js [hunk 2]
```diff
@@ -186,6 +186,8 @@ module.exports = {
                 "competitive-programming/max-number-from-array",
                 "competitive-programming/reverse-of-string",
                 "competitive-programming/diameter-of-BT",
+                "competitive-programming/generate-subsequences",
+
```
Comment (atarax665) [inline sidebars.js:190]: Remove this extra blank line.

**Comment by gayatri517** [comment]
@atarax665 i have made the changes, kindly review them

**Pull request merged**